common/notification/notification.ts

```diff
@@ -303,4 +314,57 @@ function diff(prev: any, curr: any, depth = 0) {
     return result;
 }
 
+export async function importMessageTranslations(messageTranslations: MessageTranslationFromDb[], overwrite = false) {
+    messageTranslations.sort((a, b) => a.id - b.id);
+    if (process.env.ENV !== 'dev' && overwrite) {
+        throw new Error('Cannot overwrite productive configuration');
+    }
+    let log = `Import Log ${new Date().toLocaleString('en-US')}\n`;
+
+    try {
+        await prisma.$transaction(async (prisma) => {
+            if (overwrite) {
+                const removed = await prisma.message_translation.deleteMany({});
+                log += `Through Overwrite ${removed.count} existing MessageTranslations were removed\n`;
+            }
+            const untouched = await prisma.message_translation.count({
+                where: { id: { notIn: messageTranslations.map((it) => it.id) } },
+            });
+
+            log += `${untouched} existing MessageTranslations will not be modified\n`;
+
+            for await (const translation of messageTranslations) {
+                const templateExists = await prisma.message_translation.findFirst({ where: { id: translation.id } });
+
+                if (templateExists) {
+                    const { id, ...update } = translation;
+                    await prisma.message_translation.update({
```

Comment:
What about using `prisma.message_translation.upsert` instead of separate create / update branches?

Reply:
Good point 👍🏻 Haven't thought of this. But this would lead to bigger changes for logging and stuff. We couldn't easily distinguish if it was an update or a creation and this might be helpful for debugging. I also want to keep the to functions `importNotifications` and `importMessageTranslations` relatively in sync with each other, so that they are processing everything in "almost" the same way. 

[In this discussion](https://github.com/prisma/prisma/discussions/3432) I found a way how to distinguish, but I think that we should keep the right id's used in production. So, this approach doesn't work for us.